packages/backend/src/server/NodeinfoServerService.ts

```diff
@@ -96,6 +96,11 @@ export class NodeinfoServerService {
 				metadata: {
 					nodeName: meta.name,
 					nodeDescription: meta.description,
+					nodeAdmins: [{
+						name: meta.maintainerName,
+						email: meta.maintainerEmail,
+					}],
+					// deprecated
```

Comment:
Can you please provide a source?

Reply:
if used internally its not deprecated else we should do so.

I try to stream line the nodeinfo across different federated projects.
the endgame is to release a new nodeinfo spec ... but that will take it's time.

If there is interest just visit https://github.com/jhass/nodeinfo/issues/70 ...